app/views/posts/index.html.erb

```diff
@@ -1,6 +1,33 @@
-<h2>List of posts for User <%= @user.name %></h2>
-<ul>
+<div class="posts-container">
+  <div class="user-header">
+    <div class="user-image">
+      <%= image_tag @user.photo, alt: @user.name, class: "profile-photo" %>
+    </div>
+    <div class="user-info">
+      <h2><%= @user.name %></h2>
+      <p>Number of posts: <%= @user.posts_counter %></p>
+    </div>
+  </div>
+
   <% @posts.each do |post| %>
-    <li><%= link_to post.title, user_post_path(@user, post) %></li>
+    <div class="post-card">
+      <h3><%= post.title %></h3>
+      <p><%= post.body.truncate(100) %></p>
+      <div class="post-stats">
+        <span>Comments: <%= post.comments_counter %></span>
+        <span>Likes: <%= post.likes_counter %></span>
+      </div>
+      <div class="post-comments">
+        <% post.comments.take(5).each do |comment| %>
+          <div class="comment">
+            <strong><%= comment.user.name %>:</strong> <%= comment.content %>
+          </div>
+        <% end %>
```

Comment:
- [ ] As mentioned by the previous reviewer please show the five most recent comments under the relevant post when we are on `users/:id/posts` page and you can easily do that by making use of the methods in your post model `recent_comments`. Example snippet: 

```rb
 <% post.recent_comments.each do |comment| %>
  <p><%= comment.author.name %>: <%= comment.text %></p> 
 <% end %>

```

WireFrame| your App
--|--
<img width="449" alt="Screenshot 2023-10-13 at 7 29 50 PM" src="https://github.com/cancelei/blog-in-rails/assets/44798044/b69632ff-1826-4555-b933-fe0a7160ca03">|<img width="575" alt="Screenshot 2023-10-13 at 7 30 46 PM" src="https://github.com/cancelei/blog-in-rails/assets/44798044/8045e80a-8a0f-4cd7-9ef8-98ac88995af1">